packages/react-native-bpk-component-button/src/BpkBorderedButton.android.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React, { type Element, type ElementProps, type Node } from 'react';
+import { Platform, StyleSheet, View } from 'react-native';
+import BpkIcon from 'react-native-bpk-component-icon';
+import BpkTouchableNativeFeedback from 'react-native-bpk-component-touchable-native-feedback';
+import {
+  borderRadiusPill,
+  buttonBorderWidth,
+  spacingSm,
+  spacingMd,
+  spacingBase,
+} from 'bpk-tokens/tokens/base.react.native';
+
+type ViewProps = ElementProps<typeof View>;
+type ViewStyleProp = $PropertyType<ViewProps, 'style'>;
+
+const styles = StyleSheet.create({
+  wrapper: {
+    borderRadius: borderRadiusPill,
+    overflow: 'hidden',
+  },
+  button: {
+    alignItems: 'center',
+    borderRadius: borderRadiusPill,
+    borderWidth: buttonBorderWidth,
+    flexDirection: 'row',
+    justifyContent: 'center',
+    paddingVertical: spacingMd - buttonBorderWidth,
+    paddingHorizontal: spacingBase - spacingSm - buttonBorderWidth,
+  },
+  buttonIconTrailing: {
+    flexDirection: 'row-reverse',
+  },
+  buttonIconOnly: {
+    paddingHorizontal: spacingMd,
+  },
+});
+
+export type Props = {
+  backgroundColor: string,
+  borderColor: string,
+  children: Node,
+  disabled: boolean,
+  iconOnly: boolean,
+  iconTrailing: boolean,
+  style: ViewStyleProp,
+  title: string,
+  icon: ?Element<typeof BpkIcon>,
+};
+
+const BpkBorderedButton = (props: Props) => {
+  const {
+    backgroundColor,
+    borderColor,
+    children,
+    disabled,
+    icon,
+    iconOnly,
+    iconTrailing,
+    style: userStyle,
+    title,
+    ...rest
+  } = props;
+
+  const wrapperStyle = [styles.wrapper, userStyle];
+
+  const buttonStyle = [styles.button, { backgroundColor }, { borderColor }];
+
+  if (iconTrailing) {
+    buttonStyle.push(styles.buttonIconTrailing);
+  }
+
+  if (iconOnly) {
+    buttonStyle.push(styles.buttonIconOnly);
+  }
+
+  return (
+    <View style={wrapperStyle}>
+      <BpkTouchableNativeFeedback
+        disabled={disabled}
+        borderlessBackground={Platform.Version !== 28}
```

Comment:
Shouldn't this be a less than? Or is there something specific about version 28? If so can we have a comment for this?

Reply:
I copied this from the existing button code and had the same thought, but I figured I should leave it the same until we can investigate and find out the rationale for this.